Mindbox.ExceptionsHandling.GraphQL/GraphQLCategoryMatchErrorFilter.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using HotChocolate;
+
+namespace Mindbox.ExceptionsHandling.GraphQL;
+
+public class GraphQLCategoryMatchErrorFilter : IErrorFilter
+{
+	private readonly IExceptionCategoryMatcher _exceptionCategoryMatcher;
+
+	public GraphQLCategoryMatchErrorFilter(IExceptionCategoryMatcher exceptionCategoryMatcher)
+	{
+		_exceptionCategoryMatcher = exceptionCategoryMatcher
+			?? throw new ArgumentNullException(nameof(exceptionCategoryMatcher));
+	}
+
+	public IError OnError(IError error)
+	{
+		if (error == null)
+			throw new ArgumentNullException(nameof(error));
+		return error.Code != null
+			? error
+			: ProcessError(error);
+	}
+
+	private IError ProcessError(IError error)
+	{
+		if (error.Exception is null)
```

Comment:
```suggestion
		if (error.Exception is null or SerializationException)
```

SerializationException - это исключение из либы HotChocolate. HotChocolate его выкидывает, когда клиент присылает невалидный запрос